packages/auth-construct/src/construct.test.ts

```diff
@@ -243,7 +243,8 @@ void describe('Auth construct', () => {
   void it('creates email login mechanism with specific settings', () => {
     const app = new App();
     const stack = new Stack(app);
-    const emailBodyFunction = (code: string) => `custom email body ${code}`;
+    const emailBodyFunction = (code: () => string) =>
```

Comment:
nit: if we convert `code` from a `string` to a `function`. should we also rename `code` to `getCode`?

Reply:
let me check with Josef

Reply:
folks can name the arg something different if they'd like, but this is mostly to align with changing `link` to a function as well despite not having an argument on `code`